### ISSUE TITLE
Request for API access:  Shoeb 

### ISSUE THREAD

**Issue by itscodercamp** [sunnah-com/api#1249]
* **Please tell us about yourself (include an email address):**
  
Asslamoalekum my self Shoeb an i am n software developer from india i work as a freelancer and also practicing the Islam from 2 years Alhamdulilah i have an instagram page name Quran_humse_kya_kehta_hai url:- https://www.instagram.com/quran_humse_kya_kehta_hai/
i want this api for my website were i have to add those books for access of others users and i am building an llm model which have content all of the books data and and answer the questions only according to the data

* **Your purpose in using this API:**
  
use it in my website to get the access of all those books : )

* **API rate limits:**
  
   * **Maximum requests per second:**
   i am not sure about per second but 
   * **Maximum requests per day:**
      i can make min 20 to max 50 request per day i can use the local storage function which store the books data locally by using this it can help me to load my website fast and better

* **Is your use case better served by having an offline dump of hadith data or programmatic API access?**
  
    basically i stored the books in local storage and it heck of new translations or content every week and update the data

* **What are the languages in which would you like hadith data?**
  

specially english and urdu i have my own translator which can translate it into several language 

* **What programming language will your API client be in?**
  
basically i made my backend in python django i just need an api and data jn json formate
